.github/workflows/ci.yml

```diff
@@ -18,19 +18,18 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         node_version: [12, 14, 16]
-        # TODO: uncomment this (Vite has trouble resolving imports on Windows)
-        # include:
-        #   - os: windows-latest
-        #     node_version: 14
+        include:
+          - os: windows-latest
+            node_version: 16
       fail-fast: false
     env:
       LANG: en-us
     name: 'Test: node-${{ matrix.node_version }}, ${{ matrix.os }}'
     steps:
-      - name: Checkout
+      - name: Check out code using Git
```

Comment:
nit: checkout is one word in git speak!

Reply:
Not when it’s a verb, as I said! This is just making it consistent across CI, and you wrote it this way in another action. I’m just using your own words. Even that Atlassian page you linked says “check out” or “checking out” as a verb. Please see http://notaverb.com/checkout